src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import React from 'react';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesNumber = Math.ceil(total / perPage);
+
+  const paginated = getNumbers(1, pagesNumber);
+
+  const handleNextPage = () => {
+    if (currentPage !== pagesNumber) {
+      onPageChange(currentPage + 1);
+    }
+  };
+
+  const handlePrevPage = () => {
+    if (currentPage !== 1) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={currentPage === 1 ? 'page-item disabled' : undefined}>
```

Comment:
it's better to set '' instead of undefined
```suggestion
      <li className={currentPage === 1 ? 'page-item disabled' : ''}>
```